torch/_inductor/cudagraph_trees.py

```diff
@@ -346,6 +346,17 @@ def get_manager(
     return get_container(device_index).tree_manager
 
 
+def is_cudagraph_capture_sizes(int_key: Union[int, tuple[int, ...]]) -> bool:
+    """
+    Returns true if all dynamic shapes should be captured or the dynamic shape
+    int_key should be captured.
+    """
+    return (
+        config.triton.cudagraph_capture_sizes is None
+        or int_key in config.triton.cudagraph_capture_sizes
```

Comment:
hmmm if int_key is a tuple, are we sure this does what we want? I would have assumed this would return true

```
>>> (1,2) in (1,2,3)
False
```

Reply:
nvm I guess this is more representative of the use case

```
>>> (1,2) in ((1,2),3)
True
```